drivers/hmis/app/models/hmis/ce/opportunity.rb

```diff
@@ -64,5 +65,9 @@ class Opportunity < GrdaWarehouseBase
       scope = scope.where.not(id: exclude_ids.sort.uniq)
       scope
     }
+
+    def active?
+      status.to_sym != :closed
```

Comment:
I like this but maybe use the state helper in case the status implementation changes? (from `SimpleStateMachine`)
```suggestion
      !closed?
```